packages/@ember/-internals/glimmer/tests/integration/components/input-angle-test.js

```diff
@@ -787,20 +803,24 @@ moduleFor(
     ['@test sends an action with `<Input @key-down={{action "foo"}} />` when a key is pressed'](
       assert
     ) {
-      assert.expect(2);
```

Comment:
If you are removing the `assert.expect(2);`, better to use this pattern:

```js
let trigged = 0;

// ...
actions: {
  foo() {
    trigged++;
    // ...
  }
}

// ...

this.triggerEvent(...);

assert.equal(triggered, 1, "They action was trigged exactly once");
```

Otherwise, if the action was never trigged, the test would not fail as written